### ISSUE TITLE
Fanbox including Cloudflare issue

### ISSUE THREAD

**Issue by Lordanem** [cromachina/fanbox-bot#13]
I know you're already aware of this but after some researching this seems to have been picked up on by others using cookies to verify Fanbox/Pixiv accounts, potential fix here
https://github.com/Nandaka/PixivUtil2/pull/1345

**Comment by cromachina** [comment]
Yeah unfortunately there is not a good way to automate this because of the captcha barrier. The method I described in the readme to get the correct auth tokens is probably the best that can be done.

**Comment by cromachina** [comment]
Added additional information about user-agent in the latest commits: a4c3399a24d1ba39ff4fd0231259d11b8c27c102

**Comment by cromachina** [comment]
They seem to be increasing their threshold for detection and now it's basically impossible to run the bot for myself. I've got no solution to this.
